ores/score_caches/tests/test_redis.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+
+from ..redis import Redis
```

Comment:
maybe import this as RedisScoreCache?  It gets confusing below, where we initialize Redis with a RedisMock.

Reply:
In general I like to rename the class but for now, sure thing